src/test/java/org/minions/devfund/royrodriguez/movie/MovieDatabaseTest.java

```diff
@@ -0,0 +1,330 @@
+     */
+    @Test
+    public void testForMovieAndActorAreEmpty() {
+        final MovieDatabase movieDatabase = new MovieDatabase();
+        assertTrue(movieDatabase.getMovieList().isEmpty());
+        assertTrue(movieDatabase.getActorList().isEmpty());
+    }
+
+    /**
+     * Test setMovieList.
+     */
+    @Test
+    public void testSetMovieList() {
+        final MovieDatabase movieDatabase = new MovieDatabase();
+        final ArrayList<Movie> movies = new ArrayList<>();
+        final Movie movie = new Movie();
+        movie.setName(ALPHA);
+        movies.add(movie);
+        movieDatabase.setMovieList(movies);
+        assertEquals(movie.getName(), movieDatabase.getMovieList().get(0).getName());
+    }
+
+    /**
+     * Test setActorList.
+     */
+    @Test
+    public void testSetActorList() {
+        final MovieDatabase movieDatabase = new MovieDatabase();
+        final ArrayList<Actor> actors = new ArrayList<>();
+        final Actor actor = new Actor();
+        actor.setName(ALPHA);
+        actors.add(actor);
+        movieDatabase.setActorList(actors);
+        assertEquals(actor.getName(), movieDatabase.getActorList().get(0).getName());
+    }
+
+    /**
+     * Test getBestMovie.
+     */
+    @Test
+    public void testBestMovie() {
+        final MovieDatabase movieDatabase = new MovieDatabase();
+        final ArrayList<Movie> movies = new ArrayList<>();
+        final ArrayList<Actor> actors = new ArrayList<>();
+        final Actor actor = new Actor();
+        actor.setName(ALPHA);
+        actors.add(actor);
+        Movie movie = new Movie();
+        movie.setName(ALPHA_MOVIE);
+        final double rating = 10.0;
+        movie.setRating(rating);
+        movie.setActors(actors);
+
+        final ArrayList<Actor> actors2 = new ArrayList<>();
+        final Actor actorTwo = new Actor();
+        actorTwo.setName(ALPHA_2);
+        actors2.add(actorTwo);
+        Movie movie2 = new Movie();
+        movie2.setName(ALPHA_MOVIE_2);
+        final double rating2 = 20.0;
+        movie2.setRating(rating2);
+        movie2.setActors(actors2);
+
+        movies.add(movie);
+        movies.add(movie2);
+
+        movieDatabase.setMovieList(movies);
+
+        assertEquals(ALPHA_MOVIE_2, movieDatabase.getBestMovie());
+    }
+
+    /**
+     * Test getBestMovie.
+     */
+    @Test
+    public void testBestMovieTwo() {
+        final MovieDatabase movieDatabase = new MovieDatabase();
+        final ArrayList<Movie> movies = new ArrayList<>();
+        final ArrayList<Actor> actors = new ArrayList<>();
+        final Actor actor = new Actor();
+        actor.setName(ALPHA);
+        actors.add(actor);
+        Movie movie = new Movie();
+        movie.setName(ALPHA_MOVIE);
+        final double rating = 10.0;
+        movie.setRating(rating);
+        movie.setActors(actors);
+
+        final ArrayList<Actor> actors2 = new ArrayList<>();
+        final Actor actorTwo = new Actor();
+        actorTwo.setName("Alpha2");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use already-defined constant 'ALPHA_2' instead of duplicating its value here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)